File: untitled2/src/Product.java
import java.util.Arrays;

public class Product {
    public static void main(String[] args) {
        int[] arr = {1,2,3,4};
        int sum =1;

        int count = arr.length-1;
        for (int i = 0; i < arr.length; i++) {
            sum*=arr[count];
            count--;
        }
        int[] newArr = {sum};
        System.out.println(Arrays.toString(newArr));
    }
}

File: Untitled3/product.js
array = [10, -1, 20 ,4, 5, -9, -6];
let newArray = []
let count = 0;
for (i = 0; i < array.length; i++) {
    if (array[i] < 0){
        newArray[count] = array[i];
        count++;
    }
}
for (i = 0; i < array.length; i++) {
    if (array[i] > 0){
        newArray[count] = array[i];
        count++;
    }
}
console.log(newArray)